mfai/torch/lightning_modules/segmentation.py

```diff
@@ -81,33 +107,58 @@ def get_metrics(self) -> torch.nn.ModuleDict:
             elif self.type_segmentation == "multilabel":
                 num_labels = self.model.out_channels
 
-            metrics_dict = {
-                "acc": tm.Accuracy(
-                    task=self.type_segmentation,
-                    num_classes=num_classes,
-                    num_labels=num_labels,
-                ),
-                "f1": tm.F1Score(
-                    task=self.type_segmentation,
-                    num_classes=num_classes,
-                    average=average,
-                    num_labels=num_labels,
-                ),
-                "recall_pod": tm.Recall(
-                    task=self.type_segmentation,
-                    num_classes=num_classes,
-                    average=average,
-                    num_labels=num_labels,
-                ),
-                "precision": tm.Precision(  # Precision = 1 - FAR
-                    task=self.type_segmentation,
-                    num_classes=num_classes,
-                    average=average,
-                    num_labels=num_labels,
-                ),
-            }
-            return torch.nn.ModuleDict(metrics_dict)
+            return tm.MetricCollection(
+                [
+                    tm.Accuracy(
+                        task=self.type_segmentation,
+                        num_classes=num_classes,
+                        num_labels=num_labels,
+                    ),
+                    tm.F1Score(
+                        task=self.type_segmentation,
+                        num_classes=num_classes,
+                        average=average,
+                        num_labels=num_labels,
+                    ),
+                    tm.Recall(
+                        task=self.type_segmentation,
+                        num_classes=num_classes,
+                        average=average,
+                        num_labels=num_labels,
+                    ),
+                    tm.Precision(  # Precision = 1 - FAR
+                        task=self.type_segmentation,
+                        num_classes=num_classes,
+                        average=average,
+                        num_labels=num_labels,
+                    ),
+                ]
+            )
 
+    def build_metrics_dataframe(self) -> pd.DataFrame:
+        columns_name = list(self.list_sample_metrics[0].keys())
+        return pd.DataFrame(self.list_sample_metrics, columns=columns_name)
+
+    @rank_zero_only
+    def save_test_metrics_as_csv(self, df: pd.DataFrame) -> None:
+        if self.logger is None or self.logger.log_dir is None:
+            warnings.warn(
+                "SegmentationLightningModule.save_test_metrics_as_csv() called with no logger or no local save path."
+            )
+            return
+        path_csv = Path(self.logger.log_dir) / "metrics_test_set.csv"
+        df.to_csv(path_csv, index=False)
+        print(f"--> Metrics for all samples saved in \033[91m\033[1m{path_csv}\033[0m")
```

Comment:
It is possible to concat ANSI color escape sequences `\033[91m\033[1m` => `\033[91;1m`.
```suggestion
        print(f"--> Metrics for all samples saved in \033[91;1m{path_csv}\033[0m")  # bold red
```